packages/x-date-pickers-pro/package.json

```diff
@@ -48,7 +48,7 @@
     "@date-io/moment": "^2.15.0",
     "@mui/utils": "^5.10.3",
     "@mui/x-date-pickers": "5.0.8",
-    "@mui/x-license-pro": "5.17.11",
+    "@mui/x-license-pro": "5.17.12",
```

Comment:
After thinking about it a bit more—I think we need to release `@mui/x-date-pickers-pro` package even though it does not have any changes of it's own, but I reckon we don't want our users end up needing both `5.17.11` and `5.17.2` versions of `x-license-pro` installed if they are using both pro/premium pickers and data grid packages.

But then should we also release empty `@mui/x-date-pickers`?
Have we ever had such a release case?
What do you think @flaviendelangle @m4theushw @cherniavskii ?

Reply:
Yeah, agreed. Will bump both the pickers packages for consistency.

Reply:
Interesting!

Currently, our users can have more than one version of the `x-license-pro`  package installed if they're using different versions of `x-data-grid` and `x-date-pickers`. I believe releasing `x-date-pickers` just to bump the license package version won't solve it.

Are there any known issues with multiple versions of `x-license-pro` installed though?

Reply:
Not that I'm aware of, but if we can avoid having two versions by force if people are updating packages frequently—I'd be in favour of exercising such practice. 🤔

Reply:
>  The license key will be set only for the root package.

The license key is stored in a global state so it should be shared across the instancves